src/schemas/pricelist-add.ts

```diff
@@ -1,4 +1,6 @@
-export = {
+import jsonschema from 'jsonschema';
+
+export const addSchema: jsonschema.Schema = {
```

Comment:
You don't need to export it with a name just to add types, but it is fine.

Reply:
![useful](https://user-images.githubusercontent.com/47635037/105361909-2d6be580-5c35-11eb-9b92-ab29581fa3b5.png)
useful for future use.

Reply:
Yes I know. My point was that you don't need to export it with a name to add types.

Reply:
alright. If it's okay then I will just leave it as is.

Reply:
oh wait, actually, if you don't do this, typescript will complain.